python/graphstorm/gconstruct/construct_graph.py

```diff
@@ -646,7 +646,32 @@ def close(self):
         for proc in self.processes:
             proc.join()
 
-def process_node_data(process_confs, remap_id, num_processes):
+def convert2ext_mem(arr, ext_mem_workspace, name):
```

Comment:
Will you remove those files? Or they will be automatically removed after the process exits.

Reply:
currently, i don't remove them. let me change the code to remove it.